Xamarin.Forms.Core/Page.cs

```diff
@@ -311,7 +311,8 @@ void IPageController.SendAppearing()
 				handler(this, EventArgs.Empty);
 
 			var pageContainer = this as IPageContainer<Page>;
-			((IPageController)pageContainer?.CurrentPage)?.SendAppearing();
+			if (!(this is TabbedPage))
```

Comment:
Hi @rmarinho, can you check my comment plz.
I don't think it is the right direction to check if it is TabbedPage or not.
We can share together about it.
Best regards,
Paulin.